packages/auth/src/accessToken/routes.test.ts

```diff
@@ -202,8 +202,8 @@ describe('Access Token Routes', (): void => {
           client: TEST_CLIENT,
           jwk: testJwk
         } as ClientKey)
-      const now = new Date(new Date().getTime() + 4000)
-      jest.useFakeTimers()
+      const now = new Date(new Date().getTime() + 4000 * 1000)
+      jest.useFakeTimers({ now })
       jest.setSystemTime(now)
```

Comment:
Is `setSystemTime` necessary if you specified `now` in `useFakeTimers`?
```suggestion
      jest.useFakeTimers({ now })
```

Reply:
I don't think it is anymore in Jest 28